### DISCUSSION TITLE
Can I add `lazy-lock.json` to `.gitignore`?

### DISCUSSION THREAD

**Discussion by hopezh** [LazyVim/LazyVim#850] [Q&A] [answered]
I have a modified `lazyvim` config synced with github. 

The `lazy-lock.json` file seems to change after each plugins' update.

Can I ignore this file in git? and, will this affect my config file next time I downland it from github and install it on a different computer?

**Comment by dpetka2001** [comment]
I don't see any harm in ignoring the file. If you want to install on a different computer, you only need your configuration and `Lazy` will install everything again.

**Comment by folke** [comment, marked as answer]
The lazy-lock.json file contains the exact state of all your plugins. So it's advisable to put this in .git.
When you checkout your config on another computer, the exact same revisions of your plugins will be installed.

But you can also add it to .gitignore if you don't want that

**Comment by MattWilcox** [comment]
I think my question on whether I should or shouldn't might be... what if I've got different OS's sharing dotfiles like nvim's config? For example, I have my own dotfiles shared and one machine is OSX and another is Ubuntu... I don't know enough to be sure but "as a vibe" that makes me feel a bit dubious. It's not like they're Docker environments that are identical.
